code/modules/wod13/items/clothes/head.dm

```diff
@@ -342,3 +342,194 @@
 
 		target.dropItemToGround(H)
 		target.equip_to_slot_if_possible(src, ITEM_SLOT_HEAD)
+
+/obj/item/clothing/head/vampire/redsoft
+	name = "dark red cap"
+	desc = "It's a baseball hat in a tasteful dark red colour."
+	icon_state = "dredsoft"
+	flags_cover = HEADCOVERSEYES
+
+/obj/item/clothing/head/vampire/cowboyhat
+	name = "brown cowboy hat"
+	desc = "The prevailing style of the Old West."
+	icon_state = "cowboyhat"
+	flags_cover = HEADCOVERSEYES
+
+/obj/item/clothing/head/vampire/cowboy_black
+	name = "black cowboy hat"
+	desc = "For the villainous stranger."
+	icon_state = "cowboy_black"
+	flags_cover = HEADCOVERSEYES
+
+/obj/item/clothing/head/vampire/cowboy_wide
+	name = "wide-brimmed cowboy hat"
+	desc = "For the nameless stranger."
+	icon_state = "cowboy_wide"
+	flags_cover = HEADCOVERSEYES
+
+/obj/item/clothing/head/vampire/cowboy_white
+
+	name = "white cowboy hat"
+	desc = "For the heroic stranger."
+	icon_state = "cowboy_white"
+	flags_cover = HEADCOVERSEYES
+
+/obj/item/clothing/head/vampire/cowboy_white_wide
+	name = "white wide cowboy hat"
+	desc = "For the jovial stranger."
+	icon_state = "cowboy_white_wide"
+	flags_cover = HEADCOVERSEYES
```

Comment:
these should be a subtype of ``/obj/item/clothing/head/vampire/cowboy/armorless``